src/PresentationalComponents/ZeroState/AppZeroState.js

```diff
@@ -22,7 +22,8 @@ const NewAppZeroState = ({
     customTitle,
     appId,
     children,
-    customFetchResults
+    customFetchResults,
+    customSection
```

Comment:
Mind adding a test for this functionality? :)

Reply:
ufffff you caught me, i forgot